eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -261,4 +261,12 @@
   <suppress checks="com.azure.tools.checkstyle.checks.BlacklistedWordsCheck" files=".*[/\\]test[/\\].*" />
   <suppress checks="com.azure.tools.checkstyle.checks.BlacklistedWordsCheck" files=".*[/\\]samples[/\\].*" />
   <suppress checks="com.azure.tools.checkstyle.checks.BlacklistedWordsCheck" files=".*[/\\]implementation[/\\].*" />
+
+  <!-- Suppress the check on any implementation code and codegen classes -->
+  <suppress checks="LineLength" files=".*[/\\]textanalytics[/\\].*implementation[/\\].*"/>
+
+  <!-- Suppress these checks until CI pipeline setup -->
+  <suppress checks="MissingJavadocMethod" files=".*[/\\]textanalytics[/\\].*"/>
```

Comment:
Issue to track this - #6728

Reply:
`https://github.com/Azure/azure-sdk-for-java/issues#6500` should be `https://github.com/Azure/azure-sdk-for-java/issues/6500`